index.html

```diff
@@ -168,7 +169,7 @@ <h2>Glossary</h2>
 
 <p><dfn data-lt="canonical Unicode locale identifier|canonical tag|canonical locale|Canonical Unicode locale identifier" data-plurals="canonical unicode locale identifiers" id="def_canonical_unicode_locale_identifier" class="lint-ignore">Canonical Unicode locale identifier</dfn>. A <a href="#def_well_formed" class="termref">well-formed language tag</a> resulting from the application of the <a href="#def_unicode_locale" class="termref">Unicode locale identifier</a> canonicalization rules found in [[UAX35]]. This process converts any <a href="#def_valid" class="termref">valid</a> [[BCP47]] <a href="#def_language_tag">language tag</a> into a valid <a href="#def_unicode_locale" class="termref">Unicode locale identifier</a>.  For example, deprecated subtags or irregular grandfathered tags are replaced with their preferred value from the <a href="#def_iana_language_subtag_registry" class="termref">IANA language subtag registry</a>.</p>
 
-<p><dfn id="def_encoding" class="lint-ignore">Character encoding</dfn>. The way the <a class="termref" href="#def_coded_character_set">coded character set</a> is mapped to bytes for manipulation in a computer. Commonly referred to as just the <span class="alt_name">encoding</span>. For examples and further descriptions see <a href="https://www.w3.org/International/articles/definitions-characters/index#charsets">Character encodings: Essential concepts</a>.</p>
+<p><dfn data-lt="character encoding|character encoding form" id="def_encoding" class="lint-ignore">Character encoding</dfn> or, more formally, a <em>character encoding form</em>. The way a <a class="termref" href="#def_coded_character_set">coded character set</a> is mapped to <a>code units</a> (such as bytes) for manipulation in a computer. Commonly referred to as just the <span class="alt_name">encoding</span>. For examples and further descriptions see <a href="https://www.w3.org/International/articles/definitions-characters/index#charsets">Character encodings: Essential concepts</a>.</p>
```

Comment:
I had intentionally written that CCS maps to bytes, because they all do, and that's much simpler as an explanation. See the comment on code units below.